src/shell/token.zig

```diff
@@ -69,11 +69,11 @@ pub fn tokenize(data: *[]u8) TokenizerError![][]u8 {
                 tokenizer.quote = .none;
                 ft.mem.copyForwards(
                     u8,
-                    data.*[tokenizer.quote_offset .. i - 1],
-                    data.*[tokenizer.quote_offset + 1 .. i],
+                    data[tokenizer.quote_offset .. i - 1],
+                    data[tokenizer.quote_offset + 1 .. i],
                 );
-                ft.mem.copyForwards(u8, data.*[i - 1 .. data.*.len - 1], data.*[i + 1 .. data.*.len]);
-                @memset(data.*[data.*.len - 2 .. data.*.len], 0);
+                ft.mem.copyForwards(u8, data[i - 1 .. data.len - 1], data[i + 1 .. data.len]);
```

Comment:
```suggestion
                @memcpy(data[i - 1 .. data.len - 1], data[i + 1 .. data.len]);
```
Idk why it did'nt worked the first time we tried but it works now, trust me :eyes:

Reply:
Memcpy is undefined with overlapping regions

Reply:
Oooooh yip